### PR TITLE
Add line splitting and other linting

### PR DESCRIPTION
**Pull request by b-chu**
Adds line splitting and trailing commas similar to https://github.com/mosaicml/composer/pull/3078

### CODE REVIEW

File: llmfoundry/__init__.py [hunk 1]
```diff
@@ -13,19 +13,38 @@
...
 hf_dynamic_modules_logger = logging.getLogger(
-    'transformers.dynamic_module_utils')
+    'transformers.dynamic_module_utils',
+)
 new_files_warning_filter = SpecificWarningFilter(
-    'A new version of the following files was downloaded from')
+    'A new version of the following files was downloaded from',
+)
 
 hf_dynamic_modules_logger.addFilter(new_files_warning_filter)
```
Comment (mvpatel2000) [inline llmfoundry/__init__.py:22]: can we just call it logger and make it 1 line instead of indenting?

**Review by mvpatel2000 — COMMENTED** [review]
@b-chu should we raise min line length to be 100? it seems smaller than composer? or maybe im dumb.

I think looks good to me, would like daniel's pass too

File: llmfoundry/__init__.py (hunk 1 above)
Comment (b-chu) [reply llmfoundry/__init__.py]: Done

**Comment by b-chu** [comment]
Yes foundry is 80 but composer is 120. There are many other style inconsistencies between the two repos :) I'm going to say that's out of the scope of this PR and we should discuss how/if we want to reconcile.

**Review by mvpatel2000 — APPROVED** [review]

**Review by dakinggg — APPROVED** [review]

**Pull request merged**